### PR TITLE
Simpletooltip library changes for nudge

### PR DESCRIPTION
**Pull request by manishatporter**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,13 +3,17 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
```
Comment (ashutosh-verma) [inline build.gradle:9]: `google()` is shortcut for this.

File: build.gradle [hunk 2]
```diff
@@ -18,10 +19,16 @@ buildscript {
...
 }
 
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+subprojects {
+    group = "in.porter.douglasjunior.android-simple-tooltip"
+    version = "0.2.3"
+}
```
Comment (ambujsingh) [inline build.gradle:34]: Please remove this. Already taken care of in this [commit](https://github.com/porterin/android-simple-tooltip/commit/0116155d4278ba03f89f600b5e3271537d968ae9).

File: build.gradle [hunk 3]
```diff
@@ -18,10 +19,16 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        google()
```
Comment (ambujsingh) [inline build.gradle:22]: Why is this required?

File: build.gradle [hunk 4]
```diff
@@ -3,13 +3,14 @@
 buildscript {
     repositories {
         jcenter()
+        google()
```
Comment (ambujsingh) [inline build.gradle:6]: Is this required for the gradle android tools version 3.5.0?

File: build.gradle (hunk 3 above)
Comment (manishatporter) [reply build.gradle:22]: > Why is this required?

This was added by Android Studio as I tried to build the project.
